Login.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { StyleSheet, Button, View, SafeAreaView, Text, Alert, TouchableOpacity, ImageBackground } from 'react-native';
+
+function Separator() {
+  return <View style={styles.separator} />;
+}
+
+const image = { url: '/Users/virginialadd/projects/switty/ios/assets/run-bg.jpg' };
+
+export default function Login() {
+  return (
+    <NavigationContainer>
+    <SafeAreaView style={ styles.container }>
+      <ImageBackground source={ image }  style={ styles.image } >
+      <View>
+        <TouchableOpacity style={ styles.buttonTop } >
+          <Text style={{ fontSize: 32, color: 'white' }}>Train Today</Text>
+        </TouchableOpacity>
+      </View>
+      <Separator />
+      <View>
+        <TouchableOpacity style={ styles.buttonBottom } >
+          <Text style={{ fontSize: 32, color: 'white' }}>Run Today</Text>
+        </TouchableOpacity>
+      </View>
+      </ImageBackground>
+    </SafeAreaView>
+    </NavigationContainer>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
```

Comment:
It's interesting to see the styles in the same file as your js

Reply:
this is what svelte does as well